src/main/java/com/example/bank/account/Account.java

```diff
@@ -0,0 +1,49 @@
+package com.example.bank.account;
+
+import com.example.bank.registration.jpa.Client;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Entity
+/**  W columnNames podajemy nazwę pola, czy zmapowanej kolumny w bazie danych? */
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = "account_number"))
```

Comment:
fragment dokumentacji : 
he @UniqueConstraint annotation in Spring Boot, as used within the @Table annotation, is a way to enforce a uniqueness constraint on certain columns in a database table. The columnNames attribute in this annotation specifies which columns of the table should have this unique constraint applied.

When you write @UniqueConstraint(columnNames = "account_number"), it means that the column account_number in the associated database table must have unique values for each row. In other words, no two rows in this table can have the same value for account_number. This is a common requirement for fields like account numbers, email addresses, or user IDs, where each value must be unique to prevent duplicates.

This is particularly important for maintaining data integrity and avoiding issues like duplicate entries, which can cause significant problems in applications that rely on the uniqueness of such fields.